Automattic/newspack-sponsors
# chore: update newspack scripts, prettier

### All Submissions:






### Changes proposed in this Pull Request:

Similar to https://github.com/Automattic/newspack-theme/pull/1845 and https://github.com/Automattic/newspack-blocks/pull/1174, updates prettier, stylelint and Newspack scripts to the latest versions:

* newspack-scripts@4.0.0 was installed
* wp-prettier (aliased to prettier) were updated
* ran `npx sass-migrator module src/**/*.scss` to migrate styles (no changes made)
* ran `npm run lint:scss -- --fix` to fix any possible style errors (no changes made)
* ran `npm run lint:js -- --fix` to fix an possible JS errors (no changes made)

### How to test the changes in this Pull Request:

How to test the changes in this Pull Request:

1. Review the changes
2. Confirm all jobs pass in CI
3. Reinstall NPM fresh and run `npm run build`; confirm that no errors are thrown.

### Other information:
